test/functional/wallet_importmulti.py

```diff
@@ -119,9 +119,13 @@ def get_multisig(self):
                         CScript([OP_HASH160, witness_script, OP_EQUAL]).hex(),  # p2sh-p2wsh
                         script_to_p2sh_p2wsh(script_code))  # p2sh-p2wsh addr
 
-    def test_importmulti(self, req, success, error_code=None, error_message=None):
+    def test_importmulti(self, req, success, error_code=None, error_message=None, warnings=[]):
```

Comment:
I suggest using `warnings=None` instead to clarify that `warnings` is not meant to be remembered across calls.

Background:

```
>>> def test(i, i_arr=[]):
...     i_arr.append(i)
...     return i_arr
...
>>> test(1)
[1]
>>> test(2)
[1, 2]
>>> test(3)
[1, 2, 3]
```

Suggested alternative:

```
>>> def test(i, i_arr=None):
...     if i_arr is None:
...         i_arr=[]
...     i_arr.append(i)
...     return i_arr
...
>>> test(1)
[1]
>>> test(2)
[2]
>>> test(3)
[3]
```

Reply:
Yikes, I thought only Javascript behaved like that...